src/Validator/Constraints/EventLongerThan10MinutesSlotValidator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Validator\Constraints;
+
+use App\Dto\SlotRequest;
+use Symfony\Component\Form\Form;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+final class EventLongerThan10MinutesSlotValidator extends ConstraintValidator
+{
+    public function validate($value, Constraint $constraint): void
+    {
+        if (!$value) {
+            return;
+        }
+
+        $form = $this->context->getRoot();
+
+        if (!$form instanceof Form) {
+            return;
+        }
+
+        /** @var SlotRequest $slotRequest */
+        $slotRequest = $form->getData();
+
+        $diff = $value->diff($slotRequest->start);
+        if ($diff->i < 10 && $diff->h == 0) {
```

Comment:
Can you put the 10 minutes and 0 hour as constant to better understand what those numbers are linked to?
Also use === in check with (int) needed in front of $diff->h if necessary
And constant before the $variable, like `0 === $diff->h`